resources/zib/zib-MedicationAgreement.xml

```diff
@@ -0,0 +1,326 @@
+      </type>
+    </element>
+    <element id="MedicationRequest.modifierExtension:stopType">
+      <path value="MedicationRequest.modifierExtension" />
+      <sliceName value="stopType" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+      </type>
+    </element>
+    <element id="MedicationRequest.modifierExtension:stopType.value[x]">
+      <path value="MedicationRequest.modifierExtension.value[x]" />
+      <short value="MedicationAgreementStopType" />
+      <definition value="Stop type, the manner in which this medication is discontinued (temporary or definitive)." />
+      <alias value="MedicatieafspraakStopType" />
+      <mapping>
+        <identity value="zib-medicationagreement-v1.2-2020EN" />
+        <map value="NL-CM:9.6.19954" />
+        <comment value="MedicationAgreementStopType" />
+      </mapping>
+    </element>
+    <element id="MedicationRequest.status">
+      <path value="MedicationRequest.status" />
+      <comment value="This element is implictly mapped to the zib concepts NL-CM:9.6.19936 (PeriodOfUse) and  NL-CM:9.6.19954 (MedicationAgreementStopType).&#xD;&#xA;Unless the status is explicitly recorded, the following guidance applies:&#xD;&#xA;&#xD;&#xA;* When the PeriodOfUse start date is in the future, `.status` will usually be set to _active_.&#xD;&#xA;* When PeriodOfUse start date is in the past and PeriodOfUse end date is present and in the future, `.status` will usually be set to _active_.&#xD;&#xA;* When PeriodOfUse end date is present and in the past, `.status` will usually be set to _completed_.&#xD;&#xA;* When PeriodOfUse duration and start date are known, the end date can be deducted and the previous rules apply. &#xD;&#xA;* When MedicationAgreementStopType is _113381000146106_, `.status` will usually be set to _on-hold_.&#xD;&#xA;* When MedicationAgreementStopType is _113371000146109_, `.status` will usually be set to _on-hold_.&#xD;&#xA;* When a system is unable to infer the status from the Start and End, `.status` will be set to _unknown_. The _unknown_ code is not to be used to convey other statuses. The _unknown_ code should be used when one of the statuses applies, but the authoring system doesn't know the current state of the MedicationAgreement." />
+      <mapping>
+        <identity value="zib-medicationagreement-v1.2-2020EN" />
+        <map value="NL-CM:9.6.19936" />
+        <comment value="PeriodOfUse (implicit, main mapping is on the extensions ext-TimeInterval-Period and ext-TimeInterval-Duration)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-medicationagreement-v1.2-2020EN" />
+        <map value="NL-CM:9.6.19954" />
+        <comment value="MedicationAgreementStopType (implicit, main mapping is on StopType extension)" />
+      </mapping>
+    </element>
+    <element id="MedicationRequest.category">

```

Comment:
`.intent` is required. The zib states that this is a "proposal". Would it make sense to add some guidance that unless a more acurate code is known, _proposal_ should be used.

Reply:
The zib concept description indeed mentions the word "proposal", but I don't think you could map that context to the `intent` value _proposal_. 

_proposal_: The request is a suggestion made by someone/something that doesn't have an intention to ensure it occurs and without providing an authorization to act

Some helpful text by the FHIR spec:

>It is expected that the type of requester will be restricted for different stages of a MedicationRequest.  For example, Proposals can be created by a patient, relatedPerson, Practitioner or Device.  Plans can be created by Practitioners, Patients, RelatedPersons and Devices.  Original orders can be created by a Practitioner only.

I think _order_ makes the most sense for a default value here. The zib is about a practitioner that prescribes the medication, who should have the intention and authorization to let it happen. In addition, nn STU3 fixtures the status _order_ is used. 

Shall I make a comment that: 'Unless a more accurate code is known, _order_ should be used.'?

Reply:
Mm, wait, maybe it is more a proposal or plan... ande the DispenseRequest an order.

Reply:
Created #127 to get the best possible guidance. For now we can leave it.